v0/util/error.js

```diff
@@ -31,7 +31,12 @@ function ErrorBuilder() {
   };
 
   this.setStatTags = arg => {
-    this.err.statTags = arg;
+    this.err.statTags = {
+      destType: arg.destType || arg.destination,
+      stage: arg.stage,
+      scope: arg.scope,
+      meta: arg.meta
+    };
```

Comment:
Add any residual tags too.

```suggestion
    const { destType, destination, ...rest } = arg;
    this.err.statTags = {
      destType: destType || destination,
      ...rest
    };
```

Reply:
But do we want to do this change as already existing code is more descriptive of how we want to send the stat tags, what fields we are expecting and all right ?

Reply:
The caller would give out that information. If we are concerned about incomplete tags sent from the caller, then we should add validation for the required tags.

Reply:
Are we thinking of throwing errors if the required tags are not being sent ?

Reply:
Not throw errors, but set some defaults maybe.

Reply:
This is just setting up somethings, if someone doesn't set any values it would have to handled at the outer scope. Again couple of defaults doesn't make sense.

Reply:
Ok. How about this?
```
    const { destType, destination, stage, scope, meta, ...rest } = arg;
    this.err.statTags = {
      destType: destType || destination,
      stage,
      scope,
      meta,
      ...rest
    };
```